Dizzy/Assembly.swift

```diff
@@ -36,6 +36,8 @@ class Assembly {
         container.autoregister(HomeCoordinatorType.self, argument: UIWindow.self, initializer: HomeCoordinator.init)
         container.autoregister(DiscoveryCoordinatorType.self, initializer: DiscoveryCoordinator.init)
         container.autoregister(ConversationsCoordinatorType.self, initializer: ConversationsCoordinator.init)
+        container.autoregister(WebServiceType.self, initializer: FirebaseWebService.init)
+        container.autoregister(WebServiceDispatcher.self, argument: [WebServiceType].self, initializer: WebServiceDispatcher.init)
```

Comment:
you should create protocol called "WebServiceDispatcherType" and register "WebServiceDispatcher" with it